kolko.py

```diff
@@ -0,0 +1,66 @@
+
+def printer(tab):
+    'print table with positions to choose'    
+    print('{}\n{}\n{}'.format(tab[0],tab[1],tab[2]))
+
+def instert_to_tab(player,n):
+    '''inserting player to table return update table
+        player - X or O
+        n - position number'''    
+    c=0
+    for arr in tab:
+        for i,pos in enumerate(arr):
+            if n==c:
+                arr[i]=player
+            c += 1
+    return tab
+
+def winner(tab):    
```

Comment:
Ta funkcja przyjmuje argument `tab` a jednocześnie używa zmiennej `way_to_win` oraz `players`. Używanie zmiennych globalnych jest generalnie bardzo złą praktyką i stosuje się je w na prawdę pojedynczych sytuacjach. W przypadku tego kodu to nie ma tutaj racji bytu.

Reply:
to jest cenna uwaga poprawie